### ISSUE TITLE
Uncaught RTCError: Transport channel closed

### ISSUE THREAD

**Issue by marcusand** [chrisguttandin/timing-provider#243]
hey, me again :)

I am getting `Uncaught RTCError: Transport channel closed` when another client exits.
Is there a way to handle that error? 
Would like to catch it in order to automatically reconnect.

**Comment by chrisguttandin** [comment]
Hi @marcusand, could you please elaborate a bit in which cases this happens. Some example code to reproduce the issue would be great.

**Comment by marcusand** [comment]
Ok, so just tested a bit. I get this with a very simple setup.
The code:

```js
import { TimingObject } from "timing-object";
import { TimingProvider } from "timing-provider";
import { setTimingsrc } from "timingsrc";

const video = document.querySelector("#video1");
const startButton = document.querySelector("#start");
const timingProvider = new TimingProvider("ws://localhost:3001");
const timingObject = new TimingObject(timingProvider);

timingObject.addEventListener("readystatechange", () => {
  if (timingObject.readyState === "open") {
    setTimingsrc(video, timingObject);
  }
});

startButton.addEventListener("click", () => {
  timingObject.update({ position: 0, velocity: 1 });
});
``` 
If i open that in two browser windows. If I reload one of them I get the error in the other one.
The error itself seems reasonable since one of the client disconnected but I would try to handle that error and I dont know how.

**Comment by chrisguttandin** [comment]
Thanks for the simplified test case. Turns out I forgot to handle errors for all subscriptions. The problem should be fixed in v4.3.21.

**Comment by marcusand** [comment]
Cool, thanks!
Seems to work after a quick test :)
